addon/mixins/route.js

```diff
@@ -217,6 +217,10 @@ export default Ember.Mixin.create({
     return false;
   },
 
+  updateModel(model, infinityModel) {
```

Comment:
I changed it to `updateInfinityModel` to avoid name collision, hope that's alright and sorry for inadvertently hiding this chain of comments.